dev/benchmarks/metrics_center/test/skiaperf_test.dart

```diff
@@ -0,0 +1,265 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+@Timeout(Duration(seconds: 3600))
+
+import 'dart:convert';
+
+import 'package:metrics_center/src/common.dart';
+import 'package:metrics_center/src/flutter.dart';
+import 'package:metrics_center/src/skiaperf.dart';
+
+import 'common.dart';
+
+void main() {
+  const double kValue1 = 1.0;
+  const double kValue2 = 2.0;
+
+  const String kFrameworkRevision1 = '9011cece2595447eea5dd91adaa241c1c9ef9a33';
+  const String kTaskName = 'analyzer_benchmark';
+  const String kMetric1 = 'flutter_repo_batch_maximum';
+  const String kMetric2 = 'flutter_repo_watch_maximum';
+
+  final MetricPoint cocoonPointRev1Metric1 = MetricPoint(
```

Comment:
How a cocoon Point is related to skiaperf point and flutterengine point?

Reply:
Cocoon points (framework or device lab perf metrics) are not related to FlutterEnginePoint. Cocoon points should be able to convert to SkiaPerfPoint. I'll probably add more documentations when I migrate CocoonSource / BenchmarkMetricPoint in the future.